CHANGES.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+## v1.3 - 2015-03-23
+
 * Port to Cesium 1.7.1 and Ol 3.3.0
 
 # Breaking changes
```

Comment:
You'll want a different heading type for the breaking changes (e.g. `###`, or just a bullet point like for previous releases).